Connectors/Revit/Speckle.Connectors.RevitShared/Bindings/SelectionBinding.cs

```diff
@@ -25,9 +25,11 @@ ITopLevelExceptionHandler topLevelExceptionHandler
     _revitIdleManager = idleManager;
     _topLevelExceptionHandler = topLevelExceptionHandler;
     // POC: we can inject the solution here
-    // TODO: Need to figure it out equivalent of SelectionChanged for Revit2020
+    // TODO: Need to figure it out equivalent of SelectionChanged for Revit2022
```

Comment:
remove POC: add ticket to DUI3 project and link here for investigate solution for 2022

Reply:
https://spockle.atlassian.net/browse/CNX-9874